### ISSUE TITLE
Is this code contains the whole network proposed in the paper? 

### ISSUE THREAD

**Issue by wosecz** [SijieSong/person_generation_spt#3]
Is the part of semantic parsing transformation included in this code? I'm sorry but I can't find it.

**Comment by SijieSong** [comment]
No, the published code only contains appearance generation. I will upload the semantic parsing transformation module later.

**Comment by venuszhou** [comment]
@SijieSong Will you consider to update codes of the semantic parsing transformation module ? Thanks.

**Comment by SijieSong** [comment]
> 
> 
> @SijieSong Will you consider to update codes of the semantic parsing transformation module ? Thanks.

Yes, but maybe later.
